src/common/components/help/help-install-snap.js

```diff
@@ -21,11 +21,17 @@ export default class HelpInstallSnap extends Component {
       <div className={styles.helpFlexWrapper}>
         <HeadingThree className={styles.helpFlexHeading}>{ headline }</HeadingThree>
         <div className={styles.helpText}>
-          <pre>
-            <code className={ styles.cli }>
+          <pre className={styles.cli}>
```

Comment:
Related issue. Copy button is showing up possibly when it shouldn't.

When build fails we show more detailed instructions on how to debug the build:

<img width="780" alt="screen shot 2017-08-17 at 09 11 05" src="https://user-images.githubusercontent.com/83575/29400125-48a5e9a8-832c-11e7-86cd-0a652b6d786c.png">

It's a multi line help, where it probably doesn't make much sense to copy it, especially that is contains some optional lines and comments.